week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -40,3 +41,11 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// there is no variable b declaration, i would add it to escape error,
+//but it is required to remove a line in this case i would also remove a declaration
+//and pass both variables to the function as parameters. i wrote this and then understood thatthisprogram is not only a function.
+//I have the only solution to declare a and b to run it.
```

Comment:
This is one solution :) A few other ways we could solve this are:
* Remove the `console.log(b);` line
* Move the `console.log` calls to live inside `f2` rather than be outside it
Can you think of when each may be the best idea?

Reply:
when i moved it inside f2 visual studio if i want to remove unreachable code. At least one point goes to "remove console.log" version